checkov/cloudformation/runner.py

```diff
@@ -35,6 +35,7 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
 
         if root_folder:
             for root, d_names, f_names in os.walk(root_folder):
+                [d_names.remove(d) for d in list(d_names) if d in ignored_directories]
```

Comment:
I would refactor the filtering logic into checkov.common or to base runner class